browser/spec.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: 'mixmax/browser'
```

Comment:
yeah, i'll add that. missed this because linting was passing in my `app` branch without that `env` setting due to `node/spec` was being used in a parent directory.